### DISCUSSION TITLE
PluginNotFoundException - missing external extensions

### DISCUSSION THREAD

**Discussion by Dubkoslove** [ckan/ckan#8242] [Q&A]
## CKAN version
2.10.0

## Describe the bug
I am trying to include ckan-next scheming extension, however when I run container on docker I receive an exception:
PluginNotFoundException: scheming_datasets

### Steps to reproduce
1. Installed the extension via the the Dockerfile.dev:
```
FROM ckan/ckan-dev:2.10.4
...
RUN  pip3 install -e "git+https://github.com/ckan/ckanext-scheming.git#egg=ckanext-scheming"
...
# Copy custom initialization scripts
COPY docker-entrypoint.d/* /docker-entrypoint.d/

# Apply any patches needed to CKAN core or any of the built extensions (not the
# runtime mounted ones)
COPY patches ${APP_DIR}/patches

RUN for d in $APP_DIR/patches/*; do \
        if [ -d $d ]; then \
            for f in `ls $d/*.patch | sort -g`; do \
                cd $SRC_DIR/`basename "$d"` && echo "$0: Applying patch $f to $SRC_DIR/`basename $d`"; patch -p1 < "$f" ; \
            done ; \
        fi ; \
    done
```

2. Added the extension to CKAN__PLUGINS in `.env` file:
`CKAN__PLUGINS="envvars image_view text_view datatables_view datastore datapusher scheming_datasets"`

### Expected behavior
Running CKAN with an extension

### Additional details
[Link for the extension](https://github.com/ckan/ckanext-scheming?tab=readme-ov-file#requirements)

**Comment by waterman** [comment]
I had the same problem. I think a new build run is required (`docker compose build`).
Maybe multiple build/restart cycles were required; not quite sure. I am also a newby.
But I have the same configuration and it seems to work.
